builder/scripts.d/45-harfbuzz.sh

```diff
@@ -29,7 +29,12 @@ ffbuild_dockerbuild() {
         -Dutilities=disabled
     )
 
-    if [[ $TARGET == win* || $TARGET == linux* ]]; then
+    if [[ $TARGET == win* ]]; then
+        myconf+=(
+            --cross-file=/cross.meson
+            -Dgdi=enabled
```

Comment:
I don't think enable gdi is correct because it does not do what we want to per harfbuzz doc: https://harfbuzz.github.io/integration-uniscribe.html

This option is only for test purposes and is not the Uniscribe-integration function and is only used in the `hb-shape` cli tool.